jnigen/example/in_app_java/src/android_utils/dartjni.h

```diff
@@ -260,3 +260,13 @@ static inline jthrowable check_exception() {
   if (exception == NULL) return NULL;
   return to_global_ref(exception);
 }
+
+FFI_PLUGIN_EXPORT intptr_t InitDartApiDL(void* data);
```

Comment:
I wonder if we should make this optional, only if we actually generate async callbacks.

using dart_api_dl.h means embedding a version, and once there is a version mismatch between the Dart VM and plugin, the plugin needs to be updated. And after it's updated, it can never be used on an older Dart VM / Flutter again.

Reply:
True, we should make sure that we don't do that if there's no async then.

Reply:
Currently users have to do it themselves. We're not doing it automatically (for example when calling `Jni.spawn`).